cmd/node/config/config.toml

```diff
@@ -914,25 +914,26 @@
     NumTotalPeers       = 3 # NumCrossShardPeers + num intra shard
     NumFullHistoryPeers = 3
 
+
 [HeartbeatV2]
     PeerAuthenticationTimeBetweenSendsInSec          = 600  # 10min TODO: change this for mainnet/devnet/testnet
     PeerAuthenticationTimeBetweenSendsWhenErrorInSec = 60   # 1min
-    PeerAuthenticationThresholdBetweenSends          = 0.1  # 10%
+    PeerAuthenticationTimeThresholdBetweenSends      = 0.1  # 10%   # max random threshold between messages
```

Comment:
is this related/connected with the above metric `PeerAuthenticationTimeBetweenSendsInSec` ? if yes, maybe specify something in the comment

Reply:
pushed